index.html

```diff
@@ -139,10 +110,10 @@ <h2 class="lato">Partner <span></span></h2>
     </ul>
   </section>
   <footer class="lato" id="laato">
-    <img src="./media/logo.png" class="logo" alt="Atlantis">
-    <p>2022 Atlantis World Cup. <br>
+    <img src="./media/logo2.png" class="logo" alt="Atlantis">
```

Comment:
Kindly make sure that all of your images are being rendered properly. The image is not appearing on both pages 😥

![image](https://user-images.githubusercontent.com/15807343/169220538-4d5f5227-cbcf-4c7b-a1ed-607827b1efc9.png)